content-src/asrouter/templates/SendToDeviceSnippet/SendToDeviceSnippet.schema.json

```diff
@@ -65,11 +65,6 @@
       "type": "string",
       "description": "Small icon that shows up before the title / text. 16x16px. SVG or PNG preferred. Grayscale."
     },
-    "scene2_email_placeholder_text": {
```

Comment:
Ah, I guess this is not needed because we have `scene2_input_placeholder`?

Reply:
Yes, the other field (the one removed) is not present in the template

https://github.com/mozmeao/snippets/blob/master/activity-stream/send-to-device.html#L25